src/EthMultiVault.sol

```diff
@@ -2091,7 +2257,7 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
     /// @dev only applies to triple vault IDs as input
     function getTripleAtoms(uint256 id) public view returns (uint256, uint256, uint256) {
         bool isCounterTriple = id > type(uint256).max / 2;
-        uint256[3] memory atomIds = isCounterTriple ? triples[type(uint256).max - id] : triples[id];
+        uint256[3] memory atomIds = isCounterTriple ? triples[getCounterIdFromTriple(id)] : triples[id];
```

Comment:
3.16 - Utilize Helper Functions More

Here, we make better use of the helper methods as a single source of truth.